### DISCUSSION TITLE
Leviton vrcz4 MultiLevel Switch implementation

### DISCUSSION THREAD

**Discussion by giachello** [zwave-js/zwave-js#4109] [General]
Hi there I have a Leviton VRCZ4 that issues multi level switch (up / down) commands as a dimmer, using Home Assistant

Z Wave-js says that deserialization is not implemented.

What is the best way to add it ? I am not familiar with the wave-js code base.

Giovanni

**Comment by AlCalzone** [comment]
There is a similar issue, requesting the same https://github.com/zwave-js/node-zwave-js/issues/4064
I'll add some pointers there.
